### DISCUSSION TITLE
Trying to change Project Settings in code but they remain unchanged

### DISCUSSION THREAD

**Discussion by BBITerbrack** [idea-statica/ideastatica-public#748] [Help Q&A] [answered]
As part of my code, I'm trying to change some fields of the project settings. Specifically a few things under Calculation/Checks.

I've found the fields in ConnectionSetup, changed it, updated the setup, and saved the project. Each time I run my code it makes the change and registers the previous change, but when I open the same file in the UI and navigate to the property in Project Settings, it's unchanged.

![image](https://github.com/user-attachments/assets/7ff01b16-b39a-43f8-a49f-b3a65f7dc7ce)

My code looks a little like this:
```
ConnectionSetup setup = await conClient.Project.GetSetupAsync(projectId);
setup.LimitPlasticStrain = 0.30;
await conClient.Project.UpdateSetupAsync(projectId, setup);
await conClient.Project.SaveProjectAsync(projectId, FILENAME);
```

Am I looking in the wrong place to change this property? Is this meant to be read-only? Am I missing some sort of "commit changes" thing?

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @BBITerbrack, no you are not missing anything. It is a bug when saving the settings in the downloaded file that we are looking at fixing now.  I have created an issue for it. #750

**Reply by BBITerbrack** [reply]
Thanks for taking the time to comment. Good luck bug hunting.
